zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -9041,6 +9070,57 @@ components:
             - COMPLETED_WITH_ERRORS
             - CANCELED
             - INCOMPLETED
+    BatchOperationItemSearchQuerySortRequest:
+      type: object
+      properties:
+        field:
+          description: The field to sort by.
+          type: string
+          enum:
+            - batchOperationId
+            - itemKey
+            - processInstanceKey
+            - state
+        order:
+          $ref: "#/components/schemas/SortOrderEnum"
+      required:
+        - field
+    BatchOperationItemSearchQuery:
+      description: Batch operation item search request.
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      properties:
+        sort:
+          description: Sort field criteria.
+          type: array
+          items:
+            $ref: "#/components/schemas/BatchOperationItemSearchQuerySortRequest"
+        filter:
+          description: The batch operation search filters.
+          allOf:
+            - $ref: "#/components/schemas/BatchOperationItemFilter"
+    BatchOperationItemFilter:
+      description: Batch operation item filter request.
+      type: object
+      properties:
+        batchOperationId:
+          description: The key (or operate legacy ID) of the batch operation.
+          type: string
+        itemKey:
+          description: The key of the item, e.g. a process instance key.
```

❓ Can we add at least one other example so it becomes distinguishable from the process instance key?
```suggestion
          description: The key of the item the batch operation was executed on, such as a process instance key or process definition key.
```